File: Day 3 : Intro to Conditional Statements/Solution.rb
N = gets.strip.to_i

if N % 2 != 0
    puts "Weird"
elsif N % 2 == 0 and N in (2..5)
    puts "Not Weird"
elsif N % 2 == 0 and N in (6..20)
    puts "Weird"
elsif N % 2 == 0 and N > 20
    puts "Not Weird"
end

File: Day 3 : Intro to Conditional Statements/Solution.py
if __name__ == '__main__':
    N = int(input().strip())
    if (N % 2 != 0):
        print("Weird")
    else:
        if (N >= 2 and N <= 5 or N > 20):
            print("Not Weird")
        elif (N >= 6 and N <= 20):
            print("Weird")
